conf/app.routes

```diff
@@ -641,3 +641,5 @@ GET        /:draftId/maintain-a-trust/information           controllers.Informat
 
 GET        /:draftId/declare-change                        controllers.DeclarationWhatNextController.onPageLoad(draftId: String)
 POST       /:draftId/declare-change                        controllers.DeclarationWhatNextController.onSubmit(draftId: String)
+
+GET        /:draftId/continue                                 controllers.AgentOverviewController.continue(draftId: String)
```

Comment:
@jassalrichy I'd suggest a url that is more specific that `/continue`.

Reply:
Amended to /maintain-saved-registration